### PR TITLE
Spring CORS Headers

### PR DESCRIPTION
**Pull request by blangley28**
Changes for the Prior Auth Client in DTR to work - almost all of the changes are adding Access-Control-Allow-Origin *

Also increased the probability of pending result for testing (since almost everything we show off has to do with pended Claims)

Covers PriorAuth 39: https://jira.mitre.org/browse/PRIORAUTH-39

### CODE REVIEW

File: src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java [hunk 1]
```diff
@@ -260,11 +260,14 @@ private Bundle processBundle(Bundle bundle) {
       }
 
       // generate random responses since not cancelling
-      switch (getRand(3)) {
+      switch (getRand(6)) {
       case 1:
         responseDisposition = "Granted";
```
Comment (garypgryan) [inline src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java]: Having a comment her about the probability for each case would be nice.   I can intuit what you're doing, but stating it explicitly is better.

Comment (blangley28) [reply src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java]: Added a comment about the probability

**Comment by garypgryan** [comment]
If you can reduce code by refactoring I think it's worth it.

**Comment by blangley28** [comment]
> Is there a way to refactor this so that constructing the mockMvc code cam be used across the methods? The block of code seems almost identical across several methods (searchBundles, searchBundlkesXML, getBundle, getBundleXML, and possibly others?
> 
> DefaultMockMvcBuilder builder = MockMvcBuilders. .......

Yes it is identical. I wanted to pull it out as well but the problem is that the @Autowired WebApplicationContext is not static. So I could not define builder nor mockMvc as a global variable. Attempting to set the values in setup results with a compilation error.

**Pull request merged**